### ISSUE TITLE
Comment to challenge 276 task #1

### ISSUE THREAD

**Issue by atschneid** [jo-37/the-bears-den#2]
Regarding [blog post](https://github.sommrey.de/the-bears-den/2024/07/05/ch-276.html#task-1):

I like the maths in you solution to challenge #1 but I don't think you got it quite right. eg, if you get two values at 1 hour each, your solution would say we could make one complete day. What I think you want is to take one from a value, and one from the value that gives 0 mod 24, like 1 and 23

**Comment by jo-37** [comment]
Thanks for your comment!

The question is how to interpret
> A complete day is defined as a time duration that is an exact multiple of 24 hours.

We have two hours that shall give a *duration*, but it is not described how. My first and only thought was to take the hours as an *interval* - where the *difference* shall be 0 mod 24. Thus the interval \[1, 1\] gives zero days - an exact multiple of 24.

So you may regard my solution as wrong or as a solution to a different interpretation of the task .

Actually, @mattneleigh came to the same conclusion in his [solution](https://github.com/manwar/perlweeklychallenge-club/blob/master/challenge-276/mattneleigh/perl/ch-1.pl).

**Comment by atschneid** [comment]
Fair enough. I see what you mean here. The example cases are perfectly ambiguous between the interpretations as well.
